proto/Networking/Requests/RequestType.proto

```diff
@@ -10,7 +10,7 @@ enum RequestType {
 	DOWNLOAD_REMOTE_CONFIG_VERSION = 7;         // Implemented
 	REGISTER_BACKGROUND_DEVICE = 8;             // Implemented
 	GET_PLAYER_DAY = 9;                         // Implemented
-	ACKNOWLEDGE_PUNISHMENT = 10;                // ???
+	ACKNOWLEDGE_PUNISHMENT = 10;                // Implemented
```

Comment:
do we have this one?

Reply:
https://github.com/Furtif/POGOProtos/blob/master/src/POGOProtos/Networking/Requests/Messages/AcknowledgePunishmentMessage.proto

Reply:
ok cool